pkgs/by-name/te/temp-throttle/package.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+let
+  version = "3.01";
+in
+stdenv.mkDerivation {
```

Comment:
Please follow this attribute ordering: `pname`, `version`, `src`, `installPhase`, `meta`.

See https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/attribute-ordering.md

You can use `nipkgs-hammer` tool to check it.